sklearn/multioutput.py

```diff
@@ -455,6 +455,10 @@ def fit(self, X, Y, **fit_params):
         self.order_ = self.order
         if self.order_ is None:
             self.order_ = np.array(range(Y.shape[1]))
+        elif isinstance(self.order_, tuple):
+            self.order_ = np.array(self.order_)
+            if sorted(self.order_) != list(range(Y.shape[1])):
+                raise ValueError("invalid order")
```

Comment:
This might be a neater consolidation of the logic.
```suggestion
        if isinstance(self.order_, tuple):
            self.order_ = np.array(self.order_)

        if self.order_ is None:
            self.order_ = np.array(range(Y.shape[1]))
```